conference/react-js/src/components/MyInfo/MyInfo.jsx

```diff
@@ -47,20 +91,67 @@ class MyInfo extends Component {
     e.preventDefault();
     this.props.editUser(this.state.user);
     this.handleOpenModal();
+    this.getUserPhoto(this.props.user.id);
   };
 
   handleChangePhoto = () => {
     this.setState({ showChangePhotoModal: true });
   };
 
+  changeProfilePhoto = (e) => {
+    e.preventDefault();
+
+    const file = e.target.files[0];
+    const photoURL = window.URL.createObjectURL(file);
+    this.setState(prevState => ({
+      user: {
+        ...prevState.user,
+        photo: photoURL,
+      },
+    }));
+  };
+
+  uploadPhotoToDB = (e) => {
+    e.preventDefault();
+
+    const choosePhotoBtn = document.querySelector('#choose-photo__btn');
+    const userPhoto = choosePhotoBtn.files[0];
+
+    const data = new FormData();
+    data.append('file', userPhoto);
+
+    if (choosePhotoBtn.files.length > 0) {
+      axios.post(uploadUserPhoto, data);
+      this.setState({ photoUpdateIsSuccessful: true, photoIsSelected: false });
```

Comment:
don't you think that `photoUpdateIsSuccessful: true` should be set after we receive response from `axios.post(uploadUserPhoto, data);`?